### DISCUSSION TITLE
Usage with online proctored testing/examination

### DISCUSSION THREAD

**Discussion by Eastern-Dream** [Atlas-OS/Atlas#1189] [Resources]
Hello,

Thought I would give my experience using AtlasOS during a proctored online exam. My institution use **Instructure with Honorlock browser add-on for Chrome**. I have taken exam like a couple times now and it has no issue with AtlasOS.

However, I still don't know what this means if your exam session were flagged for review and if any info gathered from the proctoring add-on under AtlasOS would make a difference in their review decision. Please exercise caution.

That is my success story with proctored online exam under AtlasOS. I encourage everyone else stumbling upon this discussion to share their experience as well.

**Comment by Ast3risk-ops** [comment]
Don't know the range of system access a chrome extension has, but it might have just caught the modified windows version fields.

It might also have been something else.

That said, you can't have a foolproof online exam tool, better to do them in person.
